lib/runtime/ppx_mysql_runtime.ml

```diff
@@ -196,3 +196,11 @@ let int_of_string_exn = wrap_deserializer int_of_string
 let int32_of_string_exn = wrap_deserializer Int32.of_string
 
 let int64_of_string_exn = wrap_deserializer Int64.of_string
+
+let bool_of_string_exn = function
```

Comment:
I am not sure whether it would not make more sense to implement the truthiness values of MySQL correctly, as such

```sql
SELECT 2 IS TRUE
```

Yields a truthy value.